sklearn/naive_bayes.py

```diff
@@ -460,6 +463,16 @@ def _update_class_log_prior(self, class_prior=None):
         else:
             self.class_log_prior_ = np.zeros(n_classes) - np.log(n_classes)
 
+    def _check_alpha(self):
+        if self.alpha < 0:
+            raise ValueError('Smoothing parameter alpha = %e. '
```

Comment:
We can now see that %e is clearly a bad pick. %.1e would be better and just as informative